### ISSUE TITLE
Where does it say that IE 11 is officially no longer supported?

### ISSUE THREAD

**Issue by mgifford** [uswds/uswds#4583]
The [browser support documentation](https://designsystem.digital.gov/documentation/developers/) is clear:

> We’ve designed the design system to support older and newer browsers through [progressive enhancement](https://en.wikipedia.org/wiki/Progressive_enhancement). The current major version of the design system (2.0) follows the [2% rule](https://gds.blog.gov.uk/2012/01/25/support-for-browsers/): we officially support any browser above 2% usage as observed by [analytics.usa.gov](https://analytics.usa.gov/). Currently, this means that the design system version 2.0 supports the newest versions of Chrome, Firefox, Safari, and Internet Explorer 11 and up.

All instances of **Internet Explorer** should be removed from lists like this with a clear (and delightful) notice that no versions if Internet Explorer are supported within the USWDS. 

**Note:** I'm not sure if we need to be testing for **Samsung Internet** but it now has 2.1% of traffic, so probably should now be seen as officially supported.

**Comment by aduth** [comment]
Related: #4538

**Comment by thisisdano** [comment]
We updated the documentation when we released USWDS 3.0.0
https://designsystem.digital.gov/documentation/developers/#browser-support

We are no longer officially supporting IE11, but we are not actively removing any IE11 functionality until we have a good reason to do so. So we somewhat tacitly still support IE11 for the time being. That said, as of 3.0.0, we may make updates that are not supported by IE11 as we move forward, and we'll make a note when we do so in any that release's release notes.
